sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-select_having.sql

```diff
@@ -0,0 +1,84 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- SELECT_HAVING
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql
+--
+-- This test file was converted from inputs/pgSQL/select_having.sql
+
```

Comment:
Can we add a comment like:

```
-- TODO: We should add UDFs in GROUP BY clause when [SPARK-28445] is resolved.
```

Reply:
> DIFF is correct. I added more combination of queries with udf() addition, so queries increased.
> Added the "-- TODO: We should add UDFs in GROUP BY clause when [SPARK-28445] is resolved." comment

Reply:
@shivusondur, let's avoid to add more combinations. `select_having.sql` file is being matched and ported from https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql

It's easier to review and check the diff if we match.

If we need, we can add separate test cases later in a different place but that's not what this JIRA targets.

Reply:
@HyukjinKwon